scripts/travis/configure_dev.sh

```diff
@@ -10,12 +10,11 @@ function installGo() {
     echo "Ensure Go version ${GO_VERSION} for platform ${OS_ARCH}"
     if [[ "${INSTALLED_GO_VERSION}" != "${GO_VERSION}" ]]; then
         echo "Installing go version ${GO_VERSION} to replace ${INSTALLED_GO_VERSION}"
-        if [[ "${OS_ARCH}" = "linux-arm64" || "${OS_ARCH}" = "linux-armv6l" ]]; then
-            sudo apt-get update -y
-            sudo apt-get -y install gimme
+        if [[ "${OS_ARCH}" == "linux-arm64" || "${OS_ARCH}" == "linux-armv6l" ]]; then
+            sudo apt install ruby ruby-dev
```

Comment:
don't you think that it's going a little bit too far ?
i.e. you're installing ruby, so that you can install travis, so that you could use gimme so that you could install go

maybe you should just download and install go as before ?
I'm concerned that you'll make the build process too long. Right now, the linux-armv6l already takes 6+ hours. Adding these would clearly make it longer..

Reply:
@tsachiherman simplified downloading gimme.